### PR TITLE
Update/add some flex config example files using the new insert()

### PR DESCRIPTION
**Pull request by joto**

### CODE REVIEW

File: flex-config/geometries-using-insert.lua [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+function osm2pgsql.process_way(object)
+    if clean_tags(object.tags) then
+--        return
+    end
+
+    -- A closed way that also has the right tags for an area is a polygon.
+    if object.is_closed and has_area_tags(object.tags) then
+        -- Creating the polygon geometry takes time, so we do it once here
+        -- and later store it in the table and use it to calclate the area.
```
Comment (lonvia) [inline flex-config/geometries-using-insert.lua:134]: typo: calculate

File: flex-config/generic.lua [hunk 2]
```diff
@@ -255,25 +257,25 @@ function osm2pgsql.process_relation(object)
...
         })
         return
     end
 
     if relation_type == 'boundary' or (relation_type == 'multipolygon' and object.tags.boundary) then
-        tables.boundaries:add_row({
+        tables.boundaries:insert({
             tags = object.tags,
-            geom = { create = 'line' }
+            geom = object:as_multilinestring()
```
Comment (lonvia) [inline flex-config/generic.lua:270]: Shouldn't there be a line_merge to have the same result as before?

File: flex-config/geometries-using-insert.lua [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+    -- Store boundary relations as multilinestrings
+    if relation_type == 'boundary' then
+        tables.boundaries:insert({
+            type = object:grab_tag('boundary'),
+            tags = object.tags,
+            -- For relations there is no clear definition what their geometry
+            -- is, so you have to decide on the geometry transformation you
+            -- want. In this case we want the boundary as multilinestring
+            -- and we want lines merged as much as possibl.e
```
Comment (lonvia) [inline flex-config/geometries-using-insert.lua:181]: typo final dot

File: flex-config/generic.lua (hunk 2 above)
Comment (joto) [reply flex-config/generic.lua]: fixed in new version

File: flex-config/geometries-using-insert.lua (hunk 3 above)
Comment (joto) [reply flex-config/geometries-using-insert.lua]: fixed in new version

File: flex-config/geometries-using-insert.lua (hunk 1 above)
Comment (joto) [reply flex-config/geometries-using-insert.lua]: fixed in new version

**Pull request merged**